src/components/FilterAndList.js

```diff
@@ -62,6 +68,23 @@ export default function FilterAndList({
 
   function handleDifficultyCards(event) {
     setDifficulty(event.target.value);
+    if (event.target.value === 'easy') {
+      setEasyActive(true);
+      setMediumActive(false);
+      setDifficultActive(false);
+    } else if (event.target.value === 'medium') {
```

Comment:
You could only write if :)
```suggestion
    } if (event.target.value === 'medium') {
```